patterns/text-faqs.php

```diff
@@ -0,0 +1,71 @@
+<?php
+/**
+ * Title: FAQs
+ * Slug: twentytwentyfive/text-faqs
+ * Categories: text, about, featured
+ * Keywords: faq, about, frequently asked
+ * Viewport width: 1400
+ * Description: A FAQs section with a FAQ heading and list of toggle questions and answers.
+ *
+ * @package    WordPress
+ * @subpackage Twenty_Twenty_Five
+ * @since      Twenty Twenty-Five 1.0
+ */
+
+?>
+<!-- wp:group {"metadata":{"name":"<?php echo esc_html_x( 'FAQs', 'Name of the FAQs pattern', 'twentytwentyfive' ); ?>"},"align":"wide","style":{"spacing":{"padding":{"top":"var:preset|spacing|50","bottom":"var:preset|spacing|50"},"blockGap":"var:preset|spacing|50"}},"layout":{"type":"constrained"}} -->
+<div class="wp-block-group alignwide" style="padding-top:var(--wp--preset--spacing--50);padding-bottom:var(--wp--preset--spacing--50)">
+	<!-- wp:heading {"align":"wide"} -->
+	<h2 class="wp-block-heading alignwide"><?php echo esc_html_x( 'Frequently Asked Questions', 'Heading of the FAQs', 'twentytwentyfive' ); ?></h2>
+	<!-- /wp:heading -->
+
+	<!-- wp:group {"align":"wide","style":{"spacing":{"blockGap":"var:preset|spacing|30"}},"layout":{"type":"default"}} -->
+	<div class="wp-block-group alignwide">
+		<!-- wp:details {"style":{"border":{"bottom":{"color":"var:preset|color|opacity-20","width":"1px"}},"spacing":{"padding":{"bottom":"var:preset|spacing|30"}},"typography":{"fontStyle":"normal","fontWeight":"400"}}} -->
+		<details class="wp-block-details" style="border-bottom-color:var(--wp--preset--color--opacity-20);border-bottom-width:1px;padding-bottom:var(--wp--preset--spacing--30);font-style:normal;font-weight:400">
+			<summary><?php echo esc_html_x( 'What is The Stories Book about?', 'Question in the FAQs pattern', 'twentytwentyfive' ); ?></summary>
+			<!-- wp:paragraph {"placeholder":"Type / to add a hidden block","style":{"typography":{"fontStyle":"normal","fontWeight":"300"}}} -->
```

Comment:
Please remove the placeholder for the paragraph. And please do it with the other paragraphs having this.

```suggestion
			<!-- wp:paragraph {"style":{"typography":{"fontStyle":"normal","fontWeight":"300"}}} -->
```